geomstats/geometry/riemannian_metric.py

```diff
@@ -257,8 +257,259 @@ def dist(self, point_a, point_b):
         dist = gs.sqrt(sq_dist)
         return dist
 
+    def variance(self,
```

Comment:
there is clearly a rebase missing here.

The functions variance and mean have been removed from this file in a former PR, I don't think you want to bring them back here?